ansible/bootstrap.yml

```diff
@@ -102,4 +118,4 @@
 
     - name: Running poststart books
       include_tasks: "{{ item }}"
-      loop: "{{ query('fileglob', \"/vagrant/vagrant/conf/ansible/playbooks/poststart/*\") | sort }}"
+      loop: "{{ query('fileglob', \"/vagrant/vagrant/conf/pre_bootstrap/*.yml\") | sort }}"
```

Comment:
I guess this has typo. Should be:
**post_bootstrap/*.yml** instead of **pre_bootstrap/*.yml**